massa-pos-exports/src/pos_final_state.rs

```diff
@@ -615,91 +650,3 @@ impl PoSFinalState {
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Why are these tests removed?

Reply:
The test no longer be needed when I edit the way deferred credits are handled, but it was a premature removal, sorry! See https://github.com/massalabs/massa/pull/3740

Reply:
Cheers. We can resolve this thread once that PR is brought in.

Reply:
Updated test in d7aa30b57ed4c97223a55b55a728087c559f46db